tox.ini

```diff
@@ -84,6 +84,7 @@ deps =
     # Extras
     botocore==1.3.23
     Pillow==3.4.2
+    h2==3.2.0
```

Comment:
:lipstick: 

```suggestion
    h2==3.2.0
    botocore==1.3.23
    Pillow==3.4.2
```

Reply:
Just out of curiosity -- What's the scheme that we should follow here for ordering?

Reply:
Oops… That was meant to be alphabetical :facepalm: